bindings/experimental/distrdf/python/DistRDF/__init__.py

```diff
@@ -116,6 +116,12 @@ def RunGraphs(proxies):
         concurrent.futures.wait(futures)
 
 
+def VariationsFor(action):
```

Comment:
(from above)
Then this should be called `action_proxy` instead of `action`? To avoid confusing with it being a node of type action.